workflow-controller/controller-service/src/main/java/io/hyscale/controller/activity/ValidatorActivity.java

```diff
@@ -38,7 +38,14 @@
     MISSING_DOCKER_REGISTRY_CREDENTIALS("Cannot find {} credentials. Do 'docker login {}' to continue with the deployment"),
     SERVICE_SPEC_VALIDATION_FAILED("Service spec validation failed for {}. {} "),
     PROFILE_VALIDATION_FAILED("Profile validation failed for {}. {} "),
-    CLUSTER_AUTHENTICATION_FAILED("Cluster authentication failed. Verify if cluster config is valid");
+    CLUSTER_AUTHENTICATION_FAILED("Cluster authentication failed. Verify if cluster config is valid"),
+
+    INVALID_NETWORK_TRAFFIC_RULES("Network Policy Validation failed due to Invalid Network Traffic Rules"),
+    INVALID_VALUE("Value not Valid for field : {} "),
```

Comment:
Invalid Service Spec. Traffic rules cannot be applied to a service exposed externally.